### ISSUE TITLE
Javadoc display issues on shift+tab

### ISSUE THREAD

**Issue by benmccann** [twosigma/beakerx#7177]
@rbidas I'm getting the same behavior described by @scottdraves in https://github.com/twosigma/beakerx/pull/6728#issuecomment-368939943 where the controls are covering the Javadoc

Also, I've only been able to trigger it on `new Plot`. It'd be great if it worked for libraries that I added via the `%%classpath add mvn` magic. E.g. if I try Tablesaw's `new Table` or `new DoubleColumn` it doesn't work

**Comment by scottdraves** [comment]
I agree it would be great if it could show the documentation from dynamically loaded classes, but I am not aware that the javadoc is in the jar on maven central.  If there were a standard way to find it we could do this.  If there were a nonstandard way we could add a magic to help find it.  Is https://jtablesaw.github.io/tablesaw/apidocs/index.html?overview-summary.html the best location?

**Comment by benmccann** [comment]
The javadoc usually is on Maven Central. The code, the source, and the javadoc are all in separate jars. For example: http://search.maven.org/#search%7Cgav%7C1%7Cg%3A"tech.tablesaw"%20AND%20a%3A"tablesaw-core"

Also, the most important thing to me would be auto-complete functionality. I'd like to see the list of available methods. This doesn't strictly require javadoc (though is more powerful if you can show the accompanying javadoc)

I know that Eclipse is able to show the Javadoc without downloading the javadoc jar from Maven. It only downloads and uses the source jar, which allows it to provide more functionality. I'm not sure exactly how it works, but it would be nice if beakerx were able to work in a similar fashion
